docs/source/reference/websecurity.md

```diff
@@ -2,23 +2,22 @@
 
 The **Security Overview** section helps you learn about:
 
-- the design of JupyterHub with respect to web security
 - the semi-trusted user
 - the available mitigations to protect untrusted users from each other
 - the value of periodic security audits
 
 This overview also helps you obtain a deeper understanding of how JupyterHub
-works.
+works, as well as its design with respect to web security.
```

Comment:
Can I ask why this item is moved out of the bullets? It makes sense to me for this to be the first item, rather than the last.

Reply:
It was due to me trying to use reference links in the topics and headings before. Web security's design encompasses all the page, so I've put it this way, but since I couldn't use reference links to the headings, it makes sense to be put as a first item, yes.